sdk/python/cmd/pulumi-language-python-exec

```diff
@@ -68,7 +68,9 @@ if __name__ == "__main__":
         os.chdir(args.pwd)
 
     successful = False
-    loop = asyncio.get_event_loop()
+
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
```

Comment:
This isn't quite what the original code was doing either--the call to `get_event_loop` would either get the event loop if one existed or create a new event loop if there wasn't one. I'm not sure that we _need_ that functionality--it may be that we're never on an event loop when we start and what you have here is sufficient--but I think that we should make sure we understand and document the circumstances under which we're calling these APIs.

Reply:
(I think that we used `get_event_loop` b/c we weren't totally confident of the conditions under which this code may be called)

Reply:
I understand what get_event_loop does. Thanks.

Reply:
>If there is no current event loop set in the current OS thread, the OS thread is main, and set_event_loop() has not yet been called, asyncio will create a new event loop and set it as the current one.

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop